Library/Homebrew/test/cask/cli/internal_stanza_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe Hbc::CLI::InternalStanza, :cask do
+  it "shows stanza of the Specified Cask" do
+    command = described_class.new("gpg", "with-gpg")
+    command.run
+    # TODO: check result
```

Comment:
Use this to check the output:

```ruby
expect {

}.to output("string").to_stdout
```

Also use curly braces for all `expect` calls so it reads nicer.

Reply:
Rubocop does not like curly braces:
`== /usr/local/Homebrew/Library/Homebrew/test/cask/cli/internal_stanza_spec.rb ==`
`C:  4: 12: Avoid using {...} for multi-line blocks.`

I also get the same offence if I 'brew style'  /usr/local/Homebrew/Library/Homebrew/test/cask/cli/fetch_spec.cli for example ...

Reply:
Weird, it should be ignored for `*_spec.rb` files.

Reply:
Yes, it is ignoring *_spec.rb files by default. I manually did a 'brew style /usr/local/Homebrew/Library/Homebrew/test/cask/cli/internal_stanza_spec.rb ' to specifically check the file.
OK, I will change the do ... end to { } so it reflects the way all the other spec files are written.

Reply:
Yeah, `brew style /usr/local/Homebrew/Library/Homebrew/test/cask/cli/internal_stanza_spec.rb` will assume that the Ruby file is a Formula, so it will run with a different configuration.